integration/js/script/failure-check

```diff
@@ -59,11 +59,17 @@ end
 
 puts "\nGenerating failure report\n"
 puts "========================================="
+isFailed = false
 resultFiles = Dir["./kite-allure-reports/*result.json"]
 resultFiles.each do |resultFile|
   file = File.read(resultFile)
   result = JSON.parse(file)
   if (result["status"]) == "FAILED"
+    isFailed = true
     explain_failure(result)
```

Comment:
optional: why do we need a new variable 'isFailed'? we can do exit(1) after explain_failure(result)

Reply:
I dont want to exit(1) right way. I would like to make sure we print out all the failure instead of quit right away.